tests/test_api/test_prefixed.rs

```diff
@@ -0,0 +1,99 @@
+use super::*;
+use cosmwasm_std::CanonicalAddr;
+use cw_multi_test::MockApiBech32;
+
+const HUMAN_ADDRESS: &str = "juno1h34lmpywh4upnjdg90cjf4j70aee6z8qqfspugamjp42e4q28kqsksmtyp";
+
+fn api_prefix(prefix: &'static str) -> MockApiBech32 {
+    MockApiBech32::new(prefix)
```

Comment:
I would suggest using `IntoBech32` to align with this change and test the highest level of API, but if you meant to add missing tests for `MockApi` then it's fine.

Reply:
This is exactly missing test for `MockApi`, `IntoBech32` will be tested somewhere else.